SPECS/PyYAML/PyYAML.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        YAML parser and emitter for Python
 Name:           PyYAML
-Version:        3.13
-Release:        8%{?dist}
+Version:        5.2
```

Comment:
What analysis has been done to confirm that this will not introduce any compatibility risk in the 2.0 LTS distro?

Reply:
Please read the last comment from Vera under the related dependency. The version is chosen based on the research Jon did that is referenced there.